http/handler.go

```diff
@@ -188,6 +196,10 @@ func (h *handler) Publish(ctx context.Context, topic *string, payload *[]byte) e
 		return errors.Wrap(errFailedPublishToMsgBroker, err)
 	}
 
+	if err := h.es.Publish(ctx, clientID, msg.Channel, msg.Subtopic); err != nil {
+		return errors.Wrap(errFailedPublishEvent, err)
+	}
```

Comment:
Event store should be wrapper around interfaces. I.e. it should be transparent from the adapter implementation because we are using middlewares. Use same approach we have for all the other services.